examples/2-query/public/index.css

```diff
@@ -13,6 +13,7 @@
   border-radius: 5px;
   box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1),
     0 4px 6px -2px rgba(0, 0, 0, 0.05);
+  margin-top: 15rem;
```

Comment:
Any reason for this? Mind posting a screenshot of the changes? It looks like you altered the structure of the HTML below as well, is there a compelling reason to do this?

Reply:
Dex is positioned in center of page (notice scroll position):

![image](https://user-images.githubusercontent.com/13261088/98453976-5de2ce00-2174-11eb-88d5-d3fd68428d92.png)

I think correct way to handle this is to use `overflow-y: scroll` on list.

Reply:
I've updated layout using flexbox

![image](https://user-images.githubusercontent.com/13261088/98454232-799ba380-2177-11eb-9420-243ff90e7b1b.png)